test/PowerShellEditorServices.Test/PsesHostFactory.cs

```diff
@@ -53,24 +45,23 @@ public static PsesInternalHost Create(ILoggerFactory loggerFactory)
             }
 
             HostStartupInfo testHostDetails = new(
-                "PowerShell Editor Services Test Host",
-                "Test.PowerShellEditorServices",
-                new Version("1.0.0"),
+                name: "PowerShell Editor Services Test Host",
+                profileId: "Test.PowerShellEditorServices",
+                version: new Version("1.0.0"),
                 psHost: new NullPSHost(),
-                TestProfilePaths,
+                profilePaths: TestProfilePaths,
                 featureFlags: Array.Empty<string>(),
                 additionalModules: Array.Empty<string>(),
-                initialSessionState,
+                initialSessionState: initialSessionState,
                 logPath: null,
-                (int)LogLevel.None,
+                logLevel: (int)LogLevel.None,
                 consoleReplEnabled: false,
                 usesLegacyReadLine: false,
                 bundledModulePath: BundledModulePath);
 
             var psesHost = new PsesInternalHost(loggerFactory, null, testHostDetails);
 
             // NOTE: Because this is used by constructors it can't use await.
-            // TODO: Should we actually load profiles here?
             if (psesHost.TryStartAsync(new HostStartOptions { LoadProfiles = true }, CancellationToken.None).GetAwaiter().GetResult())
```

Comment:
Hm, now I'm actually wondering if this should be false. I thought I had my answer, and that's that we [test we can load the profiles](https://github.com/PowerShell/PowerShellEditorServices/blob/f3d4bf2c6c2701d1cf60290b4abf72be09a4192a/test/PowerShellEditorServices.Test/Session/PsesInternalHostTests.cs#L110-L144) but we actually do the profile loading call explicitly. @rjmholt Do you know why this is explicitly set to true?